Return Code: 1
Stdout: b'Error processing -3: Input must be non-negative\n[]\n'
Stderr: b'Exception in thread Thread-1 (process_element):\nException in thread Thread-2 (process_element):\nException in thread Thread-5 (process_element):\nException in thread Thread-4 (process_element):\nTraceback (most recent call last):\nTraceback (most recent call last):\nTraceback (most recent call last):\nTraceback (most recent call last):\n  File "/home/mrigankp/fuzz4py/cpython/Lib/threading.py", line 1041, in _bootstrap_inner\n    self.run()\n    ~~~~~~~~^^\n  File "/home/mrigankp/fuzz4py/cpython/Lib/threading.py", line 992, in run\n    self._target(*self._args, **self._kwargs)\n    ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/home/mrigankp/fuzz4py/fuzz4py/inputs/289.py", line 40, in process_element\n    with dbm.sqlite3.open(\'test.db\', \'c\') as db:\n         ^^^^^^^^^^^\nAttributeError: module \'dbm\' has no attribute \'sqlite3\'\n  File "/home/mrigankp/fuzz4py/cpython/Lib/threading.py", line 1041, in _bootstrap_inner\n    self.run()\n    ~~~~~~~~^^\n  File "/home/mrigankp/fuzz4py/cpython/Lib/threading.py", line 1041, in _bootstrap_inner\n    self.run()\n    ~~~~~~~~^^\n  File "/home/mrigankp/fuzz4py/cpython/Lib/threading.py", line 992, in run\n    self._target(*self._args, **self._kwargs)\n    ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/home/mrigankp/fuzz4py/cpython/Lib/threading.py", line 1041, in _bootstrap_inner\n    self.run()\n    ~~~~~~~~^^\n  File "/home/mrigankp/fuzz4py/fuzz4py/inputs/289.py", line 40, in process_element\n    with dbm.sqlite3.open(\'test.db\', \'c\') as db:\n         ^^^^^^^^^^^\n  File "/home/mrigankp/fuzz4py/cpython/Lib/threading.py", line 992, in run\n    self._target(*self._args, **self._kwargs)\n    ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nAttributeError: module \'dbm\' has no attribute \'sqlite3\'\n  File "/home/mrigankp/fuzz4py/fuzz4py/inputs/289.py", line 40, in process_element\n    with dbm.sqlite3.open(\'test.db\', \'c\') as db:\n         ^^^^^^^^^^^\nAttributeError: module \'dbm\' has no attribute \'sqlite3\'\n  File "/home/mrigankp/fuzz4py/cpython/Lib/threading.py", line 992, in run\n    self._target(*self._args, **self._kwargs)\n    ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/home/mrigankp/fuzz4py/fuzz4py/inputs/289.py", line 40, in process_element\n    with dbm.sqlite3.open(\'test.db\', \'c\') as db:\n         ^^^^^^^^^^^\nAttributeError: module \'dbm\' has no attribute \'sqlite3\'\nTraceback (most recent call last):\n  File "/home/mrigankp/fuzz4py/fuzz4py/inputs/289.py", line 85, in <module>\n    main()\n    ~~~~^^\n  File "/home/mrigankp/fuzz4py/fuzz4py/inputs/289.py", line 73, in main\n    test_replacement()\n    ~~~~~~~~~~~~~~~~^^\n  File "/home/mrigankp/fuzz4py/fuzz4py/inputs/289.py", line 56, in test_replacement\n    obj2 = copy.replace(obj1)\n  File "/home/mrigankp/fuzz4py/cpython/Lib/copy.py", line 306, in replace\n    return func(obj, **changes)\nTypeError: test_replacement.<locals>.MyObject.__replace__() missing 1 required positional argument: \'other\'\n'
